src/server-fastly.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line spaced-comment
+/// <reference types="@fastly/js-compute" />
+
+/*
+ * Copyright (c) 2021 RethinkDNS and its authors.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import "./core/fastly/config.js";
+import { handleRequest } from "./core/doh.js";
+import * as system from "./system.js";
+import * as util from "./commons/util.js";
+
+import { allowDynamicBackends } from "fastly:experimental";
+
+allowDynamicBackends(true);
```

Comment:
Can this be moved to `src/fastly/config.js` in `prep` fn? Or, must this be called as early as possible.

No `fetch` calls are made prior to `prep`.